waiter/integration/waiter/token_request_test.clj

```diff
@@ -113,23 +120,19 @@
       (doseq [token tokens-to-create]
         (doseq [[_ router-url] (routers waiter-url)]
           (let [{:keys [body] :as token-response} (get-token router-url token :cookies cookies)
-                token-description (try (json/read-str (str body))
-                                       (catch Exception _
-                                         (is false (str "Failed to parse token " body))))]
+                token-description (parse-token-description body)]
             (assert-response-status token-response 200)
-            (is (= "/custom-endpoint" (token-description "health-check-url")))
-            (is (= service-id-prefix (token-description "name")))
-            (is current-user (token-description "owner"))
-            (is (not-any? #(contains? token-description %) ["cmd" "cpus" "mem" "run-as-user" "permitted-user"])
-                (str "Unexpected content in token response: " body)))
+            (is (contains? token-description :last-update-time))
+            (is (= {:deleted false :health-check-url "/custom-endpoint" :name service-id-prefix :owner (retrieve-username)}
```

Comment:
I thought the protocol was if we used keywords as keys and other data types as values, then commas were optional.